### PR TITLE
lint README and correct a broken link

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,30 @@
-Ziti SDK for Python
-===================
+# Ziti SDK for Python
```
Comment (qrkourier) [inline README.md:1]: consistent heading syntax

File: README.md [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
 
-# Installation
+## Installation
```
Comment (qrkourier) [inline README.md:3]: no more than one top-level heading1

File: README.md [hunk 3]
```diff
@@ -1,31 +1,30 @@
...
 
 Ziti SDK for Python is distributed via Python Package Index(PyPI)
 
 ```shell
-$ pip install openziti
+pip install openziti
```
Comment (qrkourier) [inline README.md:8]: no `$` unless showing output or command produces no output

File: README.md [hunk 4]
```diff
@@ -1,31 +1,30 @@
...
-# Usage
+## Usage
 
-## Identity Enrollment
+### Identity Enrollment
+
+Acquire [Ziti Enrollment Token](https://openziti.github.io/ziti/identities/overview.html)
 
-Acquire [Ziti Enrollment Token](https://openziti.github.io/ziti/identities/overview.html) 
 ```shell
```
Comment (qrkourier) [inline README.md:17]: empty lines around code fences

File: README.md [hunk 5]
```diff
@@ -1,31 +1,30 @@
...
-$ python -m openziti enroll -j <JWT file> -i <ID JSON file>
+python -m openziti enroll -j <JWT file> -i <ID JSON file>
 ```
 
+## Getting Help
 
-# Getting Help
-
-Please use these community resources for getting help. We use GitHub [issues](https://github.com/NetFoundry/ziti-sdk-py/issues) 
+Please use these community resources for getting help. We use GitHub [issues](https://github.com/openziti/ziti-sdk-py/issues)
```
Comment (qrkourier) [inline README.md:23]: broken link

File: README.md [hunk 6]
```diff
@@ -1,31 +1,30 @@
...
 for tracking bugs and feature requests and have limited bandwidth to address them.
 
 - Read the [docs](https://openziti.github.io/ziti/overview.html)
 - Join our [Developer Community](https://openziti.org)
 - Participate in discussion on [Discourse](https://openziti.discourse.group/)
 
-Copyright&copy; 2018-2022. NetFoundry, Inc.
\ No newline at end of file
+Copyright&copy; 2018-2022. NetFoundry, Inc.
```
Comment (qrkourier) [inline README.md:30]: missing EOF newline

**Review by qrkourier — COMMENTED** [review]
over to you @ekoby

**Review by ekoby — APPROVED** [review]

**Pull request merged**